src/main.jsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App.jsx';
+import './index.css';
```

Comment:
Just FYI, in modern code we usually leave out the semicolon.  The Javascript parser will insert one every time there's a newline except in special cases, and it's considered more readable to leave them out.